src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -1027,11 +1026,11 @@ def __import_kvset_from_file(client, path, yes):
     if KVSetConstants.KVSETRootElementName not in new_kvset:
         raise FileOperationError("file '{0}' is not in a valid '{1}' format.".format(path, ImportExportProfiles.KVSET))
 
-    kvset_to_import = [ConfigurationSetting(key=kv['key'],
-                                            label=kv['label'],
-                                            content_type=kv['content_type'],
-                                            value=kv['value'],
-                                            tags=kv['tags'])
+    kvset_to_import = [ConfigurationSetting(key=kv['key'] if 'key' in kv else None,
+                                            label=kv['label'] if 'label' in kv else None,
+                                            content_type=kv['content_type'] if 'content_type' in kv else None,
+                                            value=kv['value'] if 'value' in kv else None,
+                                            tags=kv['tags'] if 'tags' in kv else None)
                        for kv in new_kvset[KVSetConstants.KVSETRootElementName]]
```

Comment:
What if kv['tags'] is not a dict?

Reply:
That's a good point, we don't currently validate the type of these attributes, but we should.

Reply:
done!